code/modules/projectiles/projectile/bullets/sniper.dm

```diff
@@ -33,8 +33,7 @@
 	name =".50 penetrator bullet"
 	icon_state = "gauss"
 	name = "penetrator round"
-	damage = 60
-	movement_type = FLYING | UNSTOPPABLE
```

Comment:
Don't remove this line, it's needed for it to actually penetrate multiple atoms.